### PR TITLE
Add more default task switching gestures to freedom scientific braille displays

### PR DESCRIPTION
**Pull request by Oliver2213**
Summary: The Freedom Scientific braille display driver only provides a gesture for alt+tab by default.

Fixes: this pr adds gestures for alt shift tab (dots 1-2-5-6 with space), and windows tab (dots 2-3-4 with space).

Suggested what's new entry (section 'changes'): Add default gestures for alt shift tab and windows tab with all supported Freedom Scientific  braille displays.

### CODE REVIEW

**Comment by leonardder** [comment]
cc @bramd

**Comment by bramd** [comment]
@Oliver2213 I don't see a problem in adding these, however I've two questions:

1. How did you come up with the key assignments? Are these based on what's already implemented in other screenreaders or braille displays?
2. Could you add this keys to the user guide as well?

**Comment by Oliver2213** [comment]
I didn't think of adding them to the users guide, I'll certainly do that soon. As for how I came up with them... The command for alt+tab is space with dots 2-3-4-5, which is "t" in Braille; dots 1-2-5-6 are the reverse of that, which makes sense for alt+tab. The command for windows+tab is the "s" in Braille (which stands for switching, as in task switching); it's also nearly the same as the default alt_tab command, just take out dot 5.
I haven't checked what the equivalent commands are for other screen readers, or even if they have such. I can do so if you think it's important.

**Comment by bramd** [comment]
We already have lots of different key assignments for each braille
display. Defining a standard list of such gestures will solve that in
the future. For now, this is fine with me if it's documented in the user
guide as well.

**Review by LeonarddeR — APPROVED** [review]
Feel free to update the copyright line

**Comment by derekriemer** [comment]
Congrats on your first incubating PR!!

File: source/brailleDisplayDrivers/freedomScientific.py [hunk 1]
```diff
@@ -247,6 +247,8 @@ def script_toggleRightWizWheelAction(self,gesture):
 			"kb:control+end" : ("br(freedomScientific):dot4+dot5+dot6+brailleSpaceBar",),
 			"kb:alt" : ("br(freedomScientific):dot1+dot3+dot4+brailleSpaceBar",),
 			"kb:alt+tab" : ("br(freedomScientific):dot2+dot3+dot4+dot5+brailleSpaceBar",),
+			"kb:alt+shift+tab" : ("br(freedomScientific):dot1+dot2+dot5+dot6+brailleSpaceBar",),
+			"kb:win+tab" : ("br(freedomScientific):dot2+dot3+dot4+brailleSpaceBar",),
```
Comment (leonardder) [inline source/brailleDisplayDrivers/freedomScientific.py]: This one is wrong and breaks the driver currently. it should be windows, not win.

@Oliver2213: Could you provide a pr that fixes this?

**Review by LeonarddeR — COMMENTED** [review]
Thanks to @derekriemer for reporting this privately.

**Comment by Oliver2213** [comment]
Sure, thanks for letting me know.

**Pull request merged**